### PR TITLE
Categorize configuration properties by service in the appendix

### PR DESCRIPTION
**Pull request by saragluna**
As title.

### CODE REVIEW

File: docs/pom.xml [hunk 1]
```diff
@@ -104,6 +104,171 @@
...
+                </goals>
+                <configuration>
+                  <arguments>
+                    <argument>${project.basedir}/src/main/asciidoc/_configprops_kv_certificate.adoc</argument>
+                    <argument>spring.cloud.azure.keyvault.certificate.*</argument>
+                  </arguments>
+                </configuration>
+              </execution>
+              <execution>
+                <id>generate-servicebus-configprops</id>
```
Comment (yiliuTo) [inline docs/pom.xml:220]: need we add configuration properties for jms as well?

Comment (saragluna) [reply docs/pom.xml:220]: good point

Comment (saragluna) [reply docs/pom.xml:221]: added

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**